pkg/minikube/bootstrapper/images/repo.go

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 package images
 
 // DefaultKubernetesRepo is the default Kubernetes repository
-const DefaultKubernetesRepo = "k8s.gcr.io"
+const DefaultKubernetesRepo = "registry.k8s.io"
```

Comment:
This will break earlier versions of `kubeadm`, and invalidate earlier versions of preload ?

* https://github.com/kubernetes/minikube/pull/14768

Reply:
Gotcha, I restarted the build on your PR, seems to have failed for some reason. If the tests look good I'll merge and rebase this PR.